<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>Error 99 connecting to localhost:6379. Cannot assign requested address</title>
    <link>https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716</link>
    <description>Hello.
First of all this is my first approach to Docker and Redash. I&#39;m using Docker for Windows and I was able to configure it and pull the redash Image, but when I try to run it and open in a browser I got the following error:

`[2021-12-14 11:08:23,969] ERROR in app: Exception on / [GET]`

`Traceback (most recent call last):`

`  File &quot;/usr/local/lib/python2.7/site-packages/flask/app.py&quot;, line 1982, in wsgi_app`

`    response = self.full_dispatch_request()`

`  File &quot;/usr/local/lib/python2.7/site-packages/flask/app.py&quot;, line 1607, in full_dispatch_request`

`    self.try_trigger_before_first_request_functions()`

`  File &quot;/usr/local/lib/python2.7/site-packages/flask/app.py&quot;, line 1654, in try_trigger_before_first_request_functions`

`    func()`

`  File &quot;/app/redash/version_check.py&quot;, line 85, in reset_new_version_status`

`    latest_version = get_latest_version()`

`  File &quot;/app/redash/version_check.py&quot;, line 91, in get_latest_version`

`    return redis_connection.get(REDIS_KEY)`

`  File &quot;/usr/local/lib/python2.7/site-packages/redis/client.py&quot;, line 1264, in get`

`    return self.execute_command(&#39;GET&#39;, name)`

`  File &quot;/usr/local/lib/python2.7/site-packages/redis/client.py&quot;, line 772, in execute_command`

`    connection = pool.get_connection(command_name, **options)`

`  File &quot;/usr/local/lib/python2.7/site-packages/redis/connection.py&quot;, line 994, in get_connection`

`    connection.connect()`

`  File &quot;/usr/local/lib/python2.7/site-packages/redis/connection.py&quot;, line 497, in connect`

`    raise ConnectionError(self._error_message(e))`

`ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address.`

`[2021-12-14 11:08:24,093] ERROR in app: Exception on /favicon.ico [GET]`

`Traceback (most recent call last):`

`  File &quot;/usr/local/lib/python2.7/site-packages/flask/app.py&quot;, line 1982, in wsgi_app`

`    response = self.full_dispatch_request()`

`  File &quot;/usr/local/lib/python2.7/site-packages/flask/app.py&quot;, line 1607, in full_dispatch_request`

`    self.try_trigger_before_first_request_functions()`

`  File &quot;/usr/local/lib/python2.7/site-packages/flask/app.py&quot;, line 1654, in try_trigger_before_first_request_functions`

`    func()`

`  File &quot;/app/redash/version_check.py&quot;, line 85, in reset_new_version_status`

`    latest_version = get_latest_version()`

`  File &quot;/app/redash/version_check.py&quot;, line 91, in get_latest_version`

`    return redis_connection.get(REDIS_KEY)`

`  File &quot;/usr/local/lib/python2.7/site-packages/redis/client.py&quot;, line 1264, in get`

`    return self.execute_command(&#39;GET&#39;, name)`

`  File &quot;/usr/local/lib/python2.7/site-packages/redis/client.py&quot;, line 772, in execute_command`

`    connection = pool.get_connection(command_name, **options)`

`  File &quot;/usr/local/lib/python2.7/site-packages/redis/connection.py&quot;, line 994, in get_connection`

`    connection.connect()`

`  File &quot;/usr/local/lib/python2.7/site-packages/redis/connection.py&quot;, line 497, in connect`

`    raise ConnectionError(self._error_message(e))`

**`ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address.`**

### Technical details:

* Redash Version: latest redash image self hosted version ([https://hub.docker.com/r/redash/redash ](https://hub.docker.com/r/redash/redash))
* Browser/OS: Chrome/Windows 10

Someone could help me?

Thanks. Regards.</description>
    
    <lastBuildDate>Tue, 26 Jul 2022 17:17:46 +0000</lastBuildDate>
    <category>Support</category>
    <atom:link href="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>Error 99 connecting to localhost:6379. Cannot assign requested address</title>
        <dc:creator><![CDATA[KIT87]]></dc:creator>
        <description><![CDATA[
            <p>try to add in your env:<br>
REDASH_REDIS_URL=redis://redis:6379/0</p>
          <p><a href="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/7">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/7</link>
        <pubDate>Tue, 26 Jul 2022 17:17:46 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9716-7</guid>
        <source url="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716.rss">Error 99 connecting to localhost:6379. Cannot assign requested address</source>
      </item>
      <item>
        <title>Error 99 connecting to localhost:6379. Cannot assign requested address</title>
        <dc:creator><![CDATA[jesse]]></dc:creator>
        <description><![CDATA[
            <p>Can you share your docker-compose settings?</p>
          <p><a href="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/6">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/6</link>
        <pubDate>Thu, 17 Mar 2022 19:29:44 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9716-6</guid>
        <source url="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716.rss">Error 99 connecting to localhost:6379. Cannot assign requested address</source>
      </item>
      <item>
        <title>Error 99 connecting to localhost:6379. Cannot assign requested address</title>
        <dc:creator><![CDATA[srikanthpsarathy]]></dc:creator>
        <description><![CDATA[
            <p>Iâ€™m encountering the same error. Whats the fix ???</p>
          <p><a href="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/5">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/5</link>
        <pubDate>Sat, 26 Feb 2022 05:42:53 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9716-5</guid>
        <source url="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716.rss">Error 99 connecting to localhost:6379. Cannot assign requested address</source>
      </item>
      <item>
        <title>Error 99 connecting to localhost:6379. Cannot assign requested address</title>
        <dc:creator><![CDATA[jesse]]></dc:creator>
        <description><![CDATA[
            <p>What setting was wrong? This info can probably help future searchers <img src="https://emoji.discourse-cdn.com/twitter/pray.png?v=10" title=":pray:" class="emoji" alt=":pray:"></p>
          <p><a href="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/4">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/4</link>
        <pubDate>Tue, 21 Dec 2021 15:13:26 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9716-4</guid>
        <source url="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716.rss">Error 99 connecting to localhost:6379. Cannot assign requested address</source>
      </item>
      <item>
        <title>Error 99 connecting to localhost:6379. Cannot assign requested address</title>
        <dc:creator><![CDATA[amatteo90]]></dc:creator>
        <description><![CDATA[
            <p>Hello. Thanks for reply. Was just a wrong settings in .yml file</p>
          <p><a href="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/3">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/3</link>
        <pubDate>Tue, 21 Dec 2021 14:29:40 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9716-3</guid>
        <source url="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716.rss">Error 99 connecting to localhost:6379. Cannot assign requested address</source>
      </item>
      <item>
        <title>Error 99 connecting to localhost:6379. Cannot assign requested address</title>
        <dc:creator><![CDATA[jesse]]></dc:creator>
        <description><![CDATA[
            <p>What version of Redash are you trying to run? What does your docker-compose.yml file contain?</p>
          <p><a href="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/2">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/2</link>
        <pubDate>Tue, 14 Dec 2021 13:51:49 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9716-2</guid>
        <source url="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716.rss">Error 99 connecting to localhost:6379. Cannot assign requested address</source>
      </item>
      <item>
        <title>Error 99 connecting to localhost:6379. Cannot assign requested address</title>
        <dc:creator><![CDATA[amatteo90]]></dc:creator>
        <description><![CDATA[
            <p>Hello.<br>
First of all this is my first approach to Docker and Redash. Iâ€™m using Docker for Windows and I was able to configure it and pull the redash Image, but when I try to run it and open in a browser I got the following error:</p>
<p><code>[2021-12-14 11:08:23,969] ERROR in app: Exception on / [GET]</code></p>
<p><code>Traceback (most recent call last):</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app</code></p>
<p><code>    response = self.full_dispatch_request()</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1607, in full_dispatch_request</code></p>
<p><code>    self.try_trigger_before_first_request_functions()</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1654, in try_trigger_before_first_request_functions</code></p>
<p><code>    func()</code></p>
<p><code>  File "/app/redash/version_check.py", line 85, in reset_new_version_status</code></p>
<p><code>    latest_version = get_latest_version()</code></p>
<p><code>  File "/app/redash/version_check.py", line 91, in get_latest_version</code></p>
<p><code>    return redis_connection.get(REDIS_KEY)</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/redis/client.py", line 1264, in get</code></p>
<p><code>    return self.execute_command('GET', name)</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/redis/client.py", line 772, in execute_command</code></p>
<p><code>    connection = pool.get_connection(command_name, **options)</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/redis/connection.py", line 994, in get_connection</code></p>
<p><code>    connection.connect()</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/redis/connection.py", line 497, in connect</code></p>
<p><code>    raise ConnectionError(self._error_message(e))</code></p>
<p><code>ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address.</code></p>
<p><code>[2021-12-14 11:08:24,093] ERROR in app: Exception on /favicon.ico [GET]</code></p>
<p><code>Traceback (most recent call last):</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app</code></p>
<p><code>    response = self.full_dispatch_request()</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1607, in full_dispatch_request</code></p>
<p><code>    self.try_trigger_before_first_request_functions()</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1654, in try_trigger_before_first_request_functions</code></p>
<p><code>    func()</code></p>
<p><code>  File "/app/redash/version_check.py", line 85, in reset_new_version_status</code></p>
<p><code>    latest_version = get_latest_version()</code></p>
<p><code>  File "/app/redash/version_check.py", line 91, in get_latest_version</code></p>
<p><code>    return redis_connection.get(REDIS_KEY)</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/redis/client.py", line 1264, in get</code></p>
<p><code>    return self.execute_command('GET', name)</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/redis/client.py", line 772, in execute_command</code></p>
<p><code>    connection = pool.get_connection(command_name, **options)</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/redis/connection.py", line 994, in get_connection</code></p>
<p><code>    connection.connect()</code></p>
<p><code>  File "/usr/local/lib/python2.7/site-packages/redis/connection.py", line 497, in connect</code></p>
<p><code>    raise ConnectionError(self._error_message(e))</code></p>
<p><strong><code>ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address.</code></strong></p>
<h3>
<a name="technical-details-1" class="anchor" href="https://discuss.redash.io#technical-details-1"></a>Technical details:</h3>
<ul>
<li>Redash Version: latest redash image self hosted version (<a href="https://hub.docker.com/r/redash/redash" rel="noopener nofollow ugc">https://hub.docker.com/r/redash/redash </a>)</li>
<li>Browser/OS: Chrome/Windows 10</li>
</ul>
<p>Someone could help me?</p>
<p>Thanks. Regards.</p>
          <p><a href="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/1">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716/1</link>
        <pubDate>Tue, 14 Dec 2021 11:11:12 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9716-1</guid>
        <source url="https://discuss.redash.io/t/error-99-connecting-to-localhost-6379-cannot-assign-requested-address/9716.rss">Error 99 connecting to localhost:6379. Cannot assign requested address</source>
      </item>
  </channel>
</rss>
